package.json

```diff
@@ -16,14 +16,17 @@
     "unsafe-perm": true
   },
   "scripts": {
-    "dev": "webpack -d --progress --profile --colors --bail --",
-    "serve": "webpack-dev-server -d --inline --https --content-base troposphere/assets",
-    "build": "webpack --progress --profile --colors --bail --",
-    "watch": "webpack --progress --profile --colors --bail --watch -d --",
+    "clean": "rimraf troposphere/assets/bundles",
+    "serve": "webpack-dev-server --hot --env.webpack=development",
+    "build:dev": "npm run clean && webpack --env.webpack=development",
+    "build:prod": "npm run clean && webpack --env.webpack=production -p",
```

Comment:
I know this is a work-in-progress, but I want to note this sooner or later ... 

These scripts impact the `Makefile` as the `$(WEBPACK)` target uses it:
- https://github.com/cyverse/troposphere/blob/master/Makefile#L8
- https://github.com/cyverse/troposphere/blob/master/Makefile#L9

It could just be updated to use the `build:dev`, `build:prod` sub-command approach.